config.yml

```diff
@@ -19,12 +19,13 @@ pipeline:
   - number
   - amount-of-money
 - name: EntitySynonymMapper
-
 policies:
 - name: EmbeddingPolicy
   max_history: 10
   epochs: 20
-  batch_size: [32,64]
+  batch_size:
+  - 32
+  - 64
```

Comment:
what happened here?

Reply:
looks like a rasa x dump, should be fine